app/utils/request-logger.js

```diff
@@ -1,20 +1,48 @@
-'use strict'
+const logger = require('./logger')(__filename)
 
 module.exports = {
   logRequestStart: context => {
-    console.log(`Calling ${context.service} to ${context.description}`, context)
+    logger.info(`Calling ${context.service} to ${context.description}`, {
+      service: context.service,
+      method: context.method,
+      url: context.url,
+      description: context.description,
+      ...context.additionalLoggingFields
```

Comment:
What goes in `additionalLoggingFields` in selfservice it looks like very selfservice specific things. Not sure we need to include it here.